### ISSUE TITLE
The GUI size is wrong and cannot be resized

### ISSUE THREAD

**Issue by calanchue** [jrbudda/KerbalEngineer#4]
https://github.com/CYBUTEK/KerbalEngineer/issues/146

Actually this issue comes form original KER branch. But It seems that original KER branch doesn't work any more,  so I politely request this problem in here. If you have a little time, please look around this. This one is really serious problem for some language users.
Above issue suggest that i can solve it by changing encoding with hands. But KER overwrite xml file, so  it isn't solution. 
This issue is related to language encoding problem. Therefore you can't reproduce bug. If you send me file, i will gladly test it for you. please send file to calanchue at gmail.com

Thank for advance

**Comment by calanchue** [comment]
Thank you! I hope this will be soon resolved in next version. This will be really big help for other language users in KSP!

**Comment by jrbudda** [comment]
See if this solves your problem, you may need to delete all the xml files first, it should now create them as utf8.

[KerbalEngineer.zip](https://github.com/jrbudda/KerbalEngineer/files/1886216/KerbalEngineer.zip)

**Comment by calanchue** [comment]
sadly, the  KER you uploaded has other bug. After once launching it, I have experienced same gui issue. It seems that the bug comes from wiped setting file contents. After launching ksp and editing some contents in KER, All xml files in 'settings' folder in KER have same contents below.

`<?xml version="1.0" encoding="utf - 8"?><ArrayOfSettingItem xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">`

In other words, this version of KER removed all xml contents except encoding. Maybe you can reproduce it.

**Comment by jrbudda** [comment]
Hmm OK. Can you get the KSP.log file for the 'first' run with a new install? Maybe there is an error message that occurs when it is saving the files. I do not get blank files on my machine, they look fine to me.

What language is your Windows set to? Maybe I can reproduce it that way.

**Comment by calanchue** [comment]
Sorry, That was my own fault. your uploaded version is totally fine! 
Cause was my own mods which intend to fix the problem from outside of KER. 

Thank you for your help. may you saved a lot out people out there.
